### PR TITLE
Emi/refacto scenario

### PR DESCRIPTION
**Pull request by COSMO-EMI**

### CODE REVIEW

**Review by csm-thu — COMMENTED** [review]
One comment but otherwise LGTM :+1:

File: src/dialogs/DontAskAgainDialog/DontAskAgainDialog.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ export const DontAskAgainDialog = ({ id, open, onClose, labels, onConfirm }) =>
         />
       </DialogContent>
       <DialogActions>
+        {/* Use button1Props and button2Props to enable variants and colors */}
```
[the remark this answers is not in the document]
Comment (COSMO-EMI) [reply src/dialogs/DontAskAgainDialog/DontAskAgainDialog.js:51]: Indeed!

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: src/buttons/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js [hunk 2]
```diff
@@ -288,16 +288,17 @@ const CreateScenarioDialog = ({
         </Grid>
       </DialogContent>
       <DialogActions className={classes.dialogActions}>
-        <Button id="cancel" onClick={handleCloseDialog} color="primary">
+        {/* TODO Replace 'cancel' with 'stay connected' which is more accurate */}
```
Comment (csm-thu) [inline src/buttons/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js:291]: Not sure I see how "stay connected" is related to the scenario creation dialog :sweat_smile:

Comment (COSMO-EMI) [reply src/buttons/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js:291]: Mauvais endroit... Merci

File: src/inputs/UploadFile/UploadFile.js [hunk 3]
```diff
@@ -151,14 +164,16 @@ UploadFile.propTypes = {
...
+      label: 'string',
+      delete: 'string'
     }
    </pre>
    */
   labels: PropTypes.shape({
     button: PropTypes.string.isRequired,
     invalidFileMessage: PropTypes.string.isRequired,
     label: PropTypes.string,
+    delete: PropTypes.string.isRequired,
```
Comment (csm-thu) [inline src/inputs/UploadFile/UploadFile.js:176]: Adding a new key with `isRequired` in the `labels` prop could be considered a breaking change: a webapp with the previous format for `labels` using the new version of this component will cause errors to be printed in the browser console complaining about this missing key in `labels`.

We can prevent this breaking change by:
- removing `isRequired` in the key declaration
- adding a default fallback in the code: `title={ labels.delete || 'Delete file' }` (fallback values cannot be only defined in the `defaultProps` when `isRequired` is removed for the `delete` key in `labels`)

We will then have to clean this work-around in the next major version :slightly_smiling_face:

**Review by csm-thu — APPROVED** [review]
One breaking change to prevent if it is possible, but otherwise LGTM :+1:

File: src/inputs/UploadFile/UploadFile.js (hunk 3 above)
Comment (COSMO-EMI) [reply src/inputs/UploadFile/UploadFile.js:176]: Done

**Pull request merged**